kuwala/pipelines/google-poi/resources/categories.json

```diff
@@ -3,19 +3,28 @@
         "category": "administration",
         "tags": [
             "association_or_organization",
-            "city_hall"
+            "city_hall",
+            "department_of_transportation"
         ]
     },
     "airport": {
         "category": "airport",
         "tags": [
             "international_airport",
+            "civilian_passenger_airport",
             "airport"
         ]
     },
     "apartment": {
         "category": "apartment",
         "tags": [
+            "cc_housing_center",
+            "apartment_complex",
+            "hostel",
```

Comment:
I guess this fits better in "tourism", no?

Reply:
I was just referring to the hostel. If I marked multiple lines it'll show you. 

Maybe cc housing has something to do with campus. :D